packages/symantec_edr_cloud/manifest.yml

```
@@ -1,10 +1,10 @@
 format_version: 3.0.2
 name: symantec_edr_cloud
-title: Symantec EDR Cloud
-version: "1.4.0"
+title: Symantec EDR Cloud (deprecated)
+version: "1.5.0"
 source:
   license: Elastic-2.0
-description: Collect logs from Symantec EDR Cloud with Elastic Agent.
+description: Collect logs from Symantec EDR Cloud with Elastic Agent (deprecated).
```

For consistency with other packages, I suggest recommending what to use instead in the description.

```
% yq .description packages/*/manifest.yml | grep -i deprecated
Deprecated. Director is no longer supported.
Deprecated. Use the F5 BIG-IP package instead.
Deprecated. Use the Juniper SRX package instead.
Deprecated. Juniper NetScreen is no longer supported.
Deprecated. Netscout Arbor Peakflow SP is no longer supported.
Deprecated. Radware DefensePro Logs is no longer supported.
```

```suggestion
description: Deprecated. Use the Symantec Endpoint Security package instead.
```